### PR TITLE
Add docstring for Participants  

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes a part of #327

### CODE REVIEW

**Comment by deshraj** [comment]
@souravsingh please complete this PR ASAP since this is a trivial task and should not take more than couple of hours to complete.

File: apps/participants/models.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
 
 
 class Participant(TimeStampedModel):
-
+    """
+    Model representing the Participant of the competition. 
+    """
```
Comment (deshraj) [inline apps/participants/models.py]: @souravsingh if the docstring is of one line, then it should be like this 
"""Model representing the Participant of the competition. """ according to the PEP8 guidelines.

File: apps/participants/serializers.py [hunk 2]
```diff
@@ -7,7 +7,9 @@
 
 
 class ParticipantTeamSerializer(serializers.ModelSerializer):
-
+    """
+    Serializer class to map Participants to Teams.
+    """
```
Comment (deshraj) [inline apps/participants/serializers.py]: Similar case for this. And please do the same with all other cases.

**Comment by deshraj** [comment]
@souravsingh Can you please add docstring for all apps in the same PR?

**Comment by deshraj** [comment]
@souravsingh Please let me know when you have added docstring for all the apps.

**Comment by souravsingh** [comment]
@deshraj Do we have to add docstring for the classes too?

**Comment by deshraj** [comment]
We are mainly focused for adding doc string to the views and models file.

**Comment by deshraj** [comment]
@souravsingh it looks like you are busy somewhere else and could not work on this PR. I have assigned this issue to @RishabhJain2018. Please fix the build errors in the current PR so that I can merge.

**Comment by RishabhJain2018** [comment]
@deshraj Okay sure.

**Comment by souravsingh** [comment]
@deshraj I have made some fixes to the PR.

**Pull request merged**